llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -225,9 +237,13 @@ bool RISCVInstructionSelector::select(MachineInstr &MI) {
 
   switch (Opc) {
   case TargetOpcode::G_ANYEXT:
-  case TargetOpcode::G_TRUNC:
+  case TargetOpcode::G_PTRTOINT:
+  case TargetOpcode::G_TRUNC: {
     MI.setDesc(TII.get(TargetOpcode::COPY));
+    bool Selected = selectCopy(MI, MRI);
+    assert(Selected && "Could not select copy!");
```

Comment:
compilers might complain about unused `Selected` variable when assertion is disabled. So maybe we can `(void) Selected` or `return Selected;`.

Reply:
This piece of code has been rewritten recently. Please rebase.